portfolio_answer.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    
+    <link rel="stylesheet" href="style.css" />
+    
+
+</head>
+<body>
+  
+    <style scoped>
+        
+        /* 0 to 299 */
+        .on-the-fly-behavior {
+            background-image: url('particular_ad_small.png'); 
+        }
+        /* 300 to X */
+        @media (min-width: 300px) { /* or 301 if you want really the same as previously.  */
+            .on-the-fly-behavior {   
+                background-image: url('particular_ad.png');
+            }
+        }
+        </style>
```

Comment:
Great work so far. Kindly consider making your comment much clearer by elaborating further on the change you want to student to implement. In short, you can rewrite your comment by explaining it well. Consider the example below and let me know what you think 😊.
<hr/>

Kindly consider using `external` CSS instead of `inline` as using an `inline` comment is `time-consuming` and makes your HTML code look `unorganized` while using `external` CSS provides a more maintainable, reusable, and accessible way to style your website, while also reducing the file size and improving site performance.

Reply:
Kindly consider using external CSS instead of using inline comments. The inline comment makes your HTML look unorganized and is time-consuming while external CSS provides a more maintainable, reusable, and easily accessible code, while also reducing the file size and increasing site performance. You can check this [link ](https://www.javatpoint.com/external-css#:~:text=The%20external%20style%20sheet%20is,put%20inside%20the%20head%20section.)for more resources on using external CSS. 😄